### PR TITLE
Update type-conversion.md

### PR DESCRIPTION
**Pull request by BradleyBartlett**
# Fixed a space in formatting causing page metadata to show at top of article.

## Details

Fix typo in type-conversion.md.

### CODE REVIEW

**Comment by GuardRex** [comment]
Ref: https://docs.microsoft.com/en-us/dotnet/standard/base-types/type-conversion

File: docs/standard/base-types/type-conversion.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ ms.date: "03/30/2017"
 ms.prod: ".net"
 ms.reviewer: ""
 ms.suite: ""
-ms.technology:v
+ms.technology: v
```
Comment (svick) [inline docs/standard/base-types/type-conversion.md]: I don't think the technology should be "v".

Though since this Pull Request clearly improves the article, I think it's okay to merge it and fix that later.

Comment (GuardRex) [inline docs/standard/base-types/type-conversion.md]: Yeah ... should be `dotnet-standard`

Comment (GuardRex) [inline docs/standard/base-types/type-conversion.md]: @BradleyBartlett Do u want to set it to `dotnet-standard`? If not, no worries. We'll square it away on our end.

**Review by guardrex — APPROVED** [review]
@BradleyBartlett Thanks! We'll get this in asap. :shipit:

File: docs/standard/base-types/type-conversion.md (hunk 1 above)
Comment (BradleyBartlett) [inline docs/standard/base-types/type-conversion.md]: I just submitted another commit to update to dotnet-standard. Thank you!
